atd-toolbox/data_model/python/sql_templates/crash_reference_layer_triggers.sql

```diff
@@ -44,12 +44,12 @@ begin
                 limit 1);
         end if;
 
-        raise notice 'found location: % compared to previous location: %', new.location_id, old.location_id;
+        raise debug 'found location: % compared to previous location: %', new.location_id, old.location_id;
```

Comment:
if you want to see these in your client, you can set the log level with:

```sql
set client_min_messages to 'DEBUG';
```

i've added a todo to document this as well 👍